src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -368,6 +381,15 @@ export class CreateScenariosComponent implements OnInit {
       relativeTo: this.route,
     });
   }
+
+  goToScenario() {
+    // Updating breadcrums so when we navigate we can see it
+    this.planStateService.updateStateWithScenario(
+      this.scenarioId || null,
```

Comment:
this properties are string | null or number| null so I dont think we need to add || null

Reply:
Solved